src/tests/test_CA/README

```diff
@@ -9,7 +9,9 @@ pattern SSSD_test_cert_*.config. Additionally a matching key file
 SSSD_test_cert_key_%.pem should be added e.g. with
 
     openssl genpkey -algorithm RSA -out SSSD_test_cert_key_XYZ.pem -pkeyopt rsa_keygen_bits:2048
+    openssl genpkey -algorithm RSA -out SSSD_test_cert_key_0007.pem -pkeyopt rsa_keygen_bits:2048 -pkeyopt rsa_keygen_pubexp:0x25
 
+Cert 0007 will be created with nondefault exponent setting and later on signed with rsassapss as seen by some 3rd party CA:s.
```

Comment:
Hi,

can you break the line so that it is not longer than 80 characters? For the `openssl` command examples it is ok for the lines to be longer, but for the general text I'd prefer the limit. Additionally I would suggest to move the sentence about cert 0007 after the next paragraph so that the special case is mentioned after the common considerations.

bye,
Sumit